Avoid recursion error in inquirer (#8341)
rotki/rotki: docs/changelog.rst

=========

* :bug:`-` rotki will now detect new tokens right after finishing decoding new events.

> * :bug:`8169` Prevent a recursion querying the price of a token that has itself in the list of underlying tokens.

Reviewer: ```suggestion
* :bug:`8169` Prevent a recursion error when querying the price of a token.
```
